CONTRIBUTING.md

```diff
@@ -10,28 +10,31 @@ If you've never used GitHub before, we plan to include links to helpful document
 
 This project adheres to a [code of conduct](CODE_OF_CONDUCT.MD). By participating, you are expected to uphold this code.
 
-## Quick tasks (5 mins)
+## How to find tasks?
 
-### Triage any open issues
+Tasks can be found in the `Issues` tab of the project repository.
 
-Label(s): none
+All tasks that are open for contributions are labelled with [![help_wanted](https://img.shields.io/badge/-help_wanted-008672.svg)][link_help_wanted].
 
-Follow up on any issues that aren't on the roadmap and haven't had an update in a while. Suggest labels, if the issue hasn't been categorised.
+If this is your first time contributing to an open source project, then a gentle introduction can be found in tasks labelled [![good_first_issue](https://img.shields.io/badge/-good%20first%20issue-7057ff.svg)][link_good_first_issue]. If you need help, just ask and we'll mentor you.
 
-### Vote on focus topic issues
+## To code, or not to code
 
-Label(s): [![focus_topic](https://img.shields.io/badge/-focus_topic-efd6a7.svg)][link_focus_topic]
+If you don't write code, that's fine, seek out the tasks labelled [![no_code](https://img.shields.io/badge/-no%20code-fbca04.svg)][link_no_code].
```

Comment:
I think this could be phrased without focusing on the lack of a skill that much. For example: "non-code contributions are very important to us, you can find tasks here..."

Reply:
Yup, that's a much better way to explain the `no code` label!